pyproject.toml

```diff
@@ -1,3 +1,8 @@
 [build-system]
 requires = ['setuptools >= 40.8.0', 'wheel']
 build-backend = 'setuptools.build_meta'
+
+[tool.black]
+line-length = 127
+skip-string-normalization = true
+target-version = ['py27', 'py35', 'py36', 'py37', 'py38']
```

Comment:
Can reverse the order, I prefer to start with latest and go to smallest. Also, what do you think about making it merge friendly, and explode the list to be one version per line?

Reply:
I prefer it the other way :upside_down_face: but sure, doesn't bother me *much*.

Reply:
I prefer to focus on the new fancy versions, and prefer to forget the old ones at the end of the line 😆